examples/src/java/org/pantsbuild/example/README.md

```diff
@@ -211,6 +211,57 @@ Toolchain
 Pants uses [Ivy](http://ant.apache.org/ivy/) to resolve `jar` dependencies. To change how Pants
 resolves these, configure `resolve.ivy`.
 
+Starting release `1.4.0.dev26`, Pants added an option to pick [coursier](https://github.com/coursier/coursier)
+as the JVM 3rdparty resolver, with performance improvement being the main motivation. The goal is to retire ivy
+eventually. Example config to use coursier:
+
+    :::ini
+    # This will turn on coursier and turn off ivy.
+    [resolver]
+    resolver: coursier
+
+    [resolve.coursier]
+    # jvm option in case of large resolves
+    jvm_options: ['-Xmx4g', '-XX:MaxMetaspaceSize=256m']
+    # The opposite of resolve.ivy.soft_excludes
+    allow_global_excludes: False
+
+    [cache.resolve.coursier]
+    # In order to enable remote caching, need to relativize the classpath entries serialized.
```

Comment:
Can you show how to turn off remote caching but leave local caching?  Is that even possible on a cache-by-cache basis? I don't remember any more.

Reply:
My understanding is that `--cache-ignore` would even skip fingerprinting which coursier resolve needs, so if there is a way to still do fingerprinting but disabling cache checking, that'll be great.

Reply:
Discussed offline: if the code doesn't write to the cache, then these settings don't matter.

And I think that we should almost certainly do our first caching of #4397